module/zfs/zfs_sa.c

```diff
@@ -243,6 +248,11 @@ zfs_sa_set_xattr(znode_t *zp)
 	if (error)
 		goto out_free;
 
+	zilog = zfsvfs->z_log;
+	if (spa_feature_is_enabled(zfsvfs->z_os->os_spa,
+	    SPA_FEATURE_ZILSAXATTR) && zfs_zil_saxattr_enabled) {
```

Comment:
since you can't activate it here, I think this needs to be  `spa_feature_is_active()`

Reply:
yes.